docs/backup-restore.md

```diff
@@ -1,32 +1,85 @@
+Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
+
+**Cosmos DB**
+
+You must create a Cosmos DB account with a Continuous backup policy to create point-in-time restore (PITR).
+Please follow the [General way to create a transactionally-consistent backup](#general-way-to-create-a-transactionally-consistent-backup) section to create a backup.
+
+**DynamoDB**
+
+You must create tables with point-in-time recovery (PITR) and autoscaling in DynamoDB. Scalar DB Schema Loader enables PITR and autoscaling by default.
+Please follow the [General way to create a transactionally-consistent backup](#general-way-to-create-a-transactionally-consistent-backup) section to create a backup.
+
+
+## Restore Backup
+
+### JDBC databases
+
+You can restore the backup with your favorite way for JDBC databases.
+You can use `mysql` command in MySQL and `psql` command in PostgreSQL to achieve that. 
+If you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL, 
+you can restore to any point within the backup retention period with the automated backup feature.
+
+### Cassandra
+
+To minimize mistakes when doing restore operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
+You can restore to any point using `CLUSTER-ID` and `SNAPSHOT-ID`.
+Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
+
+### Cosmos DB
+
+To restore a backup, you must follow the [azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/restore-account-continuous-backup#restore-account-portal) and change the default consistency to `STRONG` after restoring the backup.
+You must use the midtime of the pause as a restore point.
+
+### DynamoDB
+
+You must restore tables one by one from the Amazon DynamoDB console using the following steps and use the midtime of the pause as a restore point
+
+* Restore with PITR of table A to another table B (alias table)
+* Take a backup of the restored table B (say, backup B)
+* Remove table A
+* Create a table named A with backup B
 
-Since Scalar DB server implements [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface, you can easily pause the server to stop accepting incoming requests and drain in-flight requests.
-You can use paused duration and choose one of the following ways depending on the underlining database to make your databases transactionally-consistent even after restored.
-* Take backups of your databases during the paused duration with snapshot feature.
-* Restore to a point in the paused duration with point-in-time-restore (PITR) feature.
+You must enable continuous backup and auto-scaling using the scalardb schema loader or Amazon DynamoDB console. The scalardb schema loader doesn't remake the existing tables.
```

Comment:
What is the must?
Using the schema-loader is the must? or changing the configurations is the must?
Please clarify.

Also, like I said many times, please explain why it needs to configure auto-scaling.
In this doc, we have never talked about the auto-scaling, so it is pretty weird to see it here, so you must explain.

In addition, what do you mean by`The scalardb schema loader doesn't remake the existing tables.`?
You are trying to say something like this?

configuring the continuous backup and auto-scaling is easier if you use the schema loader. Don't worry the schema loader only sets the missing configurations and doesn't recreate the schemas if the tables exist.

Reply:
Sorry for the inconvenience.
I will update it.

Reply:
Thank you for your review comments.
Document updated based on review comments.